sdl2promela/sdl/model.py

```diff
@@ -409,6 +430,7 @@ def __init__(self):
         self.iteratorName = None
 
 
+# TODO Investigate whether the parameters should actually be Expressions
```

Comment:
TODO?

Reply:
Yes, TODO. This is not assumed to be changed in the scope of this task.